README.md

```diff
@@ -364,6 +376,7 @@ if (args.help) {
   * No.
   * `minargs` aligns with the [POSIX Argument Syntax](https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html) here (ie. "Arguments are options if they begin with a hyphen delimiter")
   * `--number -2` will be parsed as `{ args: { "number": [""], "2": [""] } }`
+👍
```

Comment:
Agreed - unquoted values shouldn't be parsed as values if they start with `-`. It's pretty straightforward.